client/my-sites/stats/activity-log-item/activity-actor.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+
+export default class ActivityActor extends PureComponent {
+	static propTypes = {
+		avatarUrl: PropTypes.string.isRequired,
+		displayName: PropTypes.string,
+		login: PropTypes.string,
+		translatedRole: PropTypes.string.isRequired,
+	};
+
+	static defaultProps = {
+		avatarUrl: 'https://www.gravatar.com/avatar/0',
```

Comment:
My understanding is that a required prop should always be there, therefore it would never have to use the default value. Respectively, if a prop has a default value, it wouldn't need to be required.